src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -87,7 +87,11 @@ const FEE_ASSET_IDS = [
 /**
  * the accountIds from the wallet, not necessarily loaded
  */
-export const selectWalletAccountIds = (state: ReduxState) => state.portfolio.accountMetadata.ids
+export const selectWalletAccountIds = createDeepEqualOutputSelector(
```

Comment:
core biz logic of the PR - only use the account ids for the connected wallet - all other downstream selectors consume from these two